docs/conf.py

```diff
@@ -68,11 +115,13 @@
     "display_version": False,
 }
 
-# -- Test pipelines ----------------------------------------------------------
+# -- Test auto-generation pipelines ------------------------------------------
 
 base_path = Path(__file__).parent
 test_table_pipeline = base_path / "tools/test_table_gen.py"
 test_spec_pipeline = base_path / "tools/test_spec_gen.py"
+ext_api_pipeline = base_path / "tools/api_gen.py"
```

Comment:
Aren't you using a Sphinx module for that now?

Reply:
Yep, I included both extensions. The reason I kept both was because I was comparing ``autodoc`` with ``autosummary``, in the support of custom configurations, for example, selecting only specific classes/functions, and adding cross-reference in other files.